mindsdb/libs/phases/model_predictor/model_predictor.py

```diff
@@ -48,14 +75,16 @@ def run(self):
                         continue
 
                     actual_row = j + offset
-                    confidence = self.getConfidence(cell, confusion_matrix)
+                    # @TODO: Do we scrap this ?
+                    #confidence = self.getConfidence(cell, confusion_matrix)
```

Comment:
yes we can now scrap this as this Bayess implementation you have, is the right way to estimate this.